addons/stock/views/stock_picking_views.xml

```diff
@@ -176,6 +176,44 @@
             </field>
         </record>
 
+        <record model="ir.ui.view" id="stock_picking_kanban">
+            <field name="name">stock.picking.kanban</field>
+            <field name="model">stock.picking</field>
+            <field name="arch" type="xml">
+                <kanban class="o_kanban_mobile o_stock_picking_kanban">
```

Comment:
@msh-odoo 
same kanban view as previous one?
except for 'o_stock_picking_kanban' class ?